commons/common-metadata.md

```diff
@@ -290,6 +290,16 @@ It is STRONGLY RECOMMENDED to provide units in one of the following two formats:
 
 ### Statistics Object
 
+Statistics usually specify the range of values by providing the `minimum` and `maximum` values,
+but can optionally be accompanied by additional statistical values.
+Some additional statistical sizes are listed below,
+but the object can also be extended with other statistical sizes that are not listed below.
+For example, it could list additional coverages such as vegetation cover, land cover, etc.
+If statistics are provided in the Item Properties ([example](../examples/extended-item.json)),
+it is recommended to list the statistical sizes with a JSON Schema in the Collection Summaries
+to better describe the sizes ([example](../examples/collection.json)).
+Please note that some statistical sizes such as cloud cover have explicit fields in extensions such as the EO extension.
```

Comment:
The phrasing is not that clear on this line.
What is the intention?
Indicate explicitly that `eo:cloud_cover` *MUST* be used instead rather than creating a duplicate summary field?

Reply:
```suggestion
Please note that some statistical sizes such as cloud cover have explicit fields in other extensions such as the EO extension.
It is recommended to use the fields standardized in extensions in favor of providing them in the Statistics Object.
```

Reply:
Is this better?

Reply:
Excellent :)